arangod/Cluster/ClusterCollectionCreationInfo.cpp

```diff
@@ -68,10 +69,10 @@ VPackSlice arangodb::ClusterCollectionCreationInfo::isBuildingSlice() const {
 }
 
 bool arangodb::ClusterCollectionCreationInfo::needsBuildingFlag() const {
-    // Deactivated the smart graph check, our testing mock for coordinator side
-    // tries to get away without other servers by initially adding only 0
-    // shard collections (non-smart). We do not want to loose these test.
-    // So we will loose the more precise check for now.
+  // Deactivated the smart graph check, our testing mock for coordinator side
+  // tries to get away without other servers by initially adding only 0
+  // shard collections (non-smart). We do not want to loose these test.
+  // So we will loose the more precise check for now.
   /*
```

Comment:
why do you not just conditionally enable this check with `#ifndef ARANGODB_USE_GOOGLE_TESTS
`

Reply:
well I do not really see the benefit as our tests are always compiled with both Maintainer + GTest => it will never be active anyways.
Nevertheless I will add it.